tensorvis/tensorvisio/src/processors/tensorfield2dimport.cpp

```diff
@@ -54,16 +60,8 @@ TensorField2DImport::TensorField2DImport()
     addProperty(extents_);
```

Comment:
what about offset and dimension properties?

Reply:
Done.